Makefile

```
@@ -34,12 +36,14 @@ run-devserver: # Runs the dev-server
 
 run-offline: # Runs the dev-server in offline mode
 	docker compose up dev-server -d
-	@while ! curl -s http://localhost:8181 > /dev/null; do \
+	@printf 'Please wait until the development server is up and running...\n\n'
```

```suggestion
	@printf 'Please wait while the development server starts...\n\n'
```

Avoid giving the impression that users are supposed to know what they're waiting for (i.e. know what it looks like when the server is up and running)